domino/domino.py

```diff
@@ -1,10 +1,17 @@
 from typing import Optional, Tuple, List
```

Comment:
Changes in this file because of formatting?

I would make sure at least existing tests are passing.

While we want people to use `PyPI` version of `dominodatalab` that won't get these changes before a new release is published, there are still people installing it from Github master branch in their automation, so they will start complaining immediately if something is broken.